test/1.4/lib/T_io_memory.dml

```diff
@@ -54,9 +55,10 @@ bank d[i < 2][j < 2] {
 }
 
 bank e {
-    method io_memory_access(generic_transaction_t *memop, uint64 offset,
-                            void *aux) -> (bool) {
-        return default(memop, offset, cast(1234, void *));
+    param use_io_memory = false;
+    method transaction_access(transaction_t *t, uint64 offset,
+                              void *aux) -> (exception_type_t) {
+        return default(t, offset, cast(1234, void *));
```

Comment:
Seeing how the test is called `io_memory`, should the `transaction_access` case really be tested here? (This is a 1.4 test, not 1.2 where `param use_io_memory = false` still actually uses `io_memory`)

Reply:
it's primarily a test of how to implement io_memory on top level, as required by old PCI, and how to make function numbers and stuff work with that. One of the new things is that also transaction-based banks are supported there.